include/highfive/bits/H5Converter_misc.hpp

```diff
@@ -174,6 +175,34 @@ struct data_converter<
     size_t _dim;
 };
 
+// apply conversion to std::array
+template <typename T, std::size_t S>
+struct data_converter<
+    std::array<T, S>,
+    typename std::enable_if<(
+                         std::is_same<T, typename type_of_array<T>::type>::value)>::type> {
+    inline data_converter(std::array<T,S>& vec, DataSpace& space, size_t dim = 0)
+    : _space(&space), _dim(dim) {
+        assert(_space->getDimensions().size() >= dim);
+        (void)vec;
+    }
+    
+    inline typename type_of_array<T>::type*
+    transform_read(std::array<T, S>& vec) {
+        return &(vec[0]);
```

Comment:
Why not vec.data()?

Reply:
That's how `std::vector` was done. I changed one of them to `.data`, will do the other too.

Reply:
Done (only changed the arrays, not the vectors).